site/app/libraries/database/DatabaseQueries.php

```diff
@@ -1159,7 +1159,7 @@ public function updateGradingRegistration($user_id, $user_group, $sections) {
      * @return integer - group number of user in the given class
      */
     public function getGroupForUserInClass($semester, $course_name, $user_id) {
```

Comment:
I noticed that a lot of variable are still named $semester, will these be changed in another round of refactor?

Reply:
Yep!